docs/docs/training-data-format.mdx

```diff
@@ -685,3 +685,83 @@ rasa test
 
 If you want to know more about testing head over to
 [Testing Your Assistant](testing-your-assistant.mdx).
+
+
+## End-to-end Training
+
+:::caution experimental feature
+End-to-end training is an experimental feature.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+However, the functionality might be changed or removed in the future.
+If you have feedback (positive or negative) please share it with us on the [Rasa Forum](https://forum.rasa.com).
+
+:::
+
+With [end-to-end training](stories.mdx#end-to-end-training), you do not have to deal with the specific
+intents of the messages that are extracted by the NLU pipeline.
+Instead, you can put the text of the user message directly in the stories,
+by using `user` key.
+
+These end-to-end user messages follow the format:
+
+```yaml-rasa {4}
+stories:
+- story: user message structure
+  steps:
+    - user: the actual text of the user message
+```
+
+In addition, you can add entity tags that can be extracted
+by the [TED Policy](./policies.mdx#ted-policy).
+The syntax for entity tags is the same as in
+[the NLU training data](./training-data-format.mdx#entities).
+For example, the following story contains the user utterance
+` I can always go for sushi`. By using the syntax from the NLU training data
+`[sushi](cuisine)`, you can mark `sushi` as an entity of type `cuisine`.
+
+```yaml-rasa {4}
+stories:
+- story: story with entities
+  steps:
+  - user: I can always go for [sushi](cuisine)
+  - action: utter_suggest_cuisine
+```
+
+
+Similarly, you can put bot utterances directly in the stories,
+by using the `bot` key followed by the text that you want your bot to say.
+
+A story with only a bot utterance might look like this:
+
+```yaml-rasa {7}
+stories:
+- story: story with an end-to-end response
+  steps:
+  - intent: greet
+    entities:
+      name: Ivan
+  - bot: Hello, Ivan!
```

Comment:
I wonder if this is a good example? Isn't this a situation were you would always use an `utter_` action so you could output a slot value?

Reply:
true

Reply:
`Hello, a person with a name!` does it sound good 😅